README.md

```diff
@@ -1,8 +1,19 @@
 # Ng2-c3 Documentation
 
-    Ng2-c3 is an Angular 2 component to C3.js charting library.
-    # References
-      - c3js library [http://c3js.org] 
-      - Angular 2 https://angular.io/
-    #usage 
-      - ToDO
+  Ng2-c3 is an Angular 2 component to C3.js charting library.
+  # References
+    - c3js library [http://c3js.org] 
+    - Angular 2 https://angular.io/
+  #Build Steps
+    - npm install
+    - npm start
+    - Server would start at http://localhost:8080/webpack-dev-server/
+  #Usage Manual
+      #Example 
+          For creating a Line Chart
+          Create a Wrapper and include ng2-c3 component 
+          <div style="width:500px;height:500px">
+              <ng2-c3 bind-data="_data"></ng2-c3>
```

Comment:
If we go by the looks, we can first avoid unnecessary word ' bind'. The component way looks cleaner. Advantages we can worry later on.